measures/measures-data.json

```diff
@@ -142,6 +142,294 @@
     "objective": "healthInformationExchange"
   },
   {
+    "category": "quality",
+    "firstPerformanceYear": 2017,
+    "lastPerformanceYear": null,
+    "metricType": "float",
+    "title": "CAHPS for MIPS SSM: Getting Timely Care, Appointments and Information",
+    "description": "",
```

Comment:
The Tech Spec said that RAND would provide the `description` field for these measures later on -- should we create a JIRA ticket to do that so we don't forget?